pkg/kev/converter/kubernetes/transform_test.go

```diff
@@ -793,13 +803,20 @@ var _ = Describe("Transform", func() {
 		})
 
 		When("project service label instructing to expose the k8s service is specified as `domain.name`", func() {
```

Comment:
tiny: label -> extension. We can always fish those things out later too.

Reply:
Fixed in #525